### ISSUE TITLE
Definitions for references?

### ISSUE THREAD

**Issue by jjwtay** [epiphone/class-validator-jsonschema#46]
According to:
[classTransformer.test.ts](https://github.com/epiphone/class-validator-jsonschema/blob/master/__tests__/classTransformer.test.ts)

nested classes currently generate ref's pointing at the json-schema definitions field but doesn't auto include the definitions. Is there any plan to include those in the future? Currently I am hand adding them post schemas being generated but would be amazing if they could be added automatically during generation.

**Comment by epiphone** [comment]
Good point. There's no plan to automatically populate `definitions`, no, but I'd be happy to review a PR if somebody wants to have a go at it. We need to make sure the implementation plays well with how the library interacts with `routing-controllers-openapi` though: https://github.com/epiphone/routing-controllers-openapi#validation-classes.
